### ISSUE TITLE
Add lead-in text to connect download panel to intro panel

### ISSUE THREAD

**Issue by sarahmonster** [Automattic/components.underscores.me#35]
Since we don't have a title on the download panel anymore, we might want a little connecting text between the intro and downloads panel. Something like this, from the original mockup:

<img width="653" alt="screen shot 2015-11-21 at 16 54 39" src="https://cloud.githubusercontent.com/assets/376315/11319487/99fdd622-9070-11e5-97aa-9e149c7b7153.png">

The "get a starter theme package" bit, and a down-pointing arrow maybe to lead into the next panel, so there's a clear path between the two.

**Comment by sixhours** [comment]
I agree, some kind of call to action or graphical element is necessary to bring the focus down. 

I'm not sure the headline in the mock-up above stands out enough from the surrounding text to do that.

Maybe a simple link with an arrow, or some subtle downward animation.

**Comment by alaczek** [comment]
I really liked the "Let's get you a theme!" heading above the slider that we had in the original mockup. Placing the heading in the top section, rather then together with the slider, makes it visually disconnected.
